### PR TITLE
Even more ballistic vests

### PR DESCRIPTION
**Pull request by Dan1ss1mo**
#### Summary
Content "Even more ballistic vests"

#### Purpose of change
Adds thigh ballistic protection along with options to combine it with existing vests.

#### Describe the solution
 Changed arm encumbrance to 3, as 5 seems too much, compared to somthing like sheet metal armguards that give 7 and also added ability to attach sling to all military vests

#### Testing
WIP

#### Additional context
- "can we have mds with molle?"
- "we have mds with molle at home"
- mds with molle at home:
FAS1-scaled
ZXWhcgNWQexRPTj4oBCBB87ibm0Pi7w0kgS6GPb4afbqV9cI4y4eg4ExgveMkMLLDFm8poxJ1YDKMDuwdHSipL79

### CODE REVIEW

File: data/json/items/armor/ballistic_armor.json [hunk 1]
```diff
@@ -95,6 +95,98 @@
       }
     ]
   },
+  {
+    "id": "ballistic_vest_esapi_sling",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "ESAPI ballistic vest with sling" },
```
Comment (TheShadowFerret) [inline data/json/items/armor/ballistic_armor.json:102]:
```suggestion
    "name": { "str": "ESAPI ballistic vest with sling", "str_pl": "ESAPI ballistic vests with sling" },
```

File: data/json/items/armor/ballistic_armor.json [hunk 2]
```diff
@@ -390,7 +482,172 @@
...
+        "specifically_covers": [ "leg_hip_l", "leg_hip_r" ],
+        "volume_encumber_modifier": 0
+      }
+    ]
+  },
+  {
+    "id": "ballistic_vest_heavy_sling",
+    "type": "ARMOR",
+    "copy-from": "ballistic_vest_esapi_sling",
+    "name": { "str": "heavy ballistic vest with sling" },
```
Comment (TheShadowFerret) [inline data/json/items/armor/ballistic_armor.json:513]:
```suggestion
    "name": { "str": "heavy ballistic vest with sling", "str_pl": "heavy ballistic vests with sling" },
```

File: data/json/items/armor/ballistic_armor.json [hunk 3]
```diff
@@ -390,7 +482,172 @@
...
+        "volume_encumber_modifier": 0
+      }
+    ]
+  },
+  {
+    "id": "ballistic_vest_heavier_sling",
+    "type": "ARMOR",
+    "copy-from": "ballistic_vest_heavy_sling",
+    "name": { "str": "Full Ballistic Protection" },
+    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  This configuration adds shoulder protectors, as well as protection for the neck and groin.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/ballistic_armor.json:628]:
```suggestion
    "name": { "str": "Full Ballistic Protection with sling" },
    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  This configuration adds shoulder protectors, as well as protection for the neck and groin.  This one has a single point sling wrapped around the shoulders using a system to keep your weapon extra secure.",
```

Where ma thighs + the plural on this is going to make no sense

File: data/json/items/armor/ballistic_armor.json [hunk 4]
```diff
@@ -390,7 +482,172 @@
...
+        "volume_encumber_modifier": 0
+      }
+    ]
+  },
+  {
+    "id": "ballistic_vest_heavier",
+    "type": "ARMOR",
+    "copy-from": "ballistic_vest_heavy",
+    "name": { "str": "Full Ballistic Protection" },
+    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  This configuration adds shoulder protectors, as well as protection for the neck and groin.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/ballistic_armor.json:565]: The plural on this won't make sense + no mention of thigh protectors.

File: data/json/items/armor/ballistic_armor.json [hunk 5]
```diff
@@ -439,7 +710,46 @@
...
+        "specifically_covers": [ "arm_shoulder_l", "arm_shoulder_r", "arm_upper_l", "arm_upper_r" ],
+        "volume_encumber_modifier": 0
+      }
+    ]
+  },
+  {
+    "id": "ballistic_vest_heavy_arms_sling",
+    "type": "ARMOR",
+    "copy-from": "ballistic_vest_heavy_sling",
+    "name": { "str": "legless heavy ballistic vest with sling" },
```
Comment (TheShadowFerret) [inline data/json/items/armor/ballistic_armor.json:729]:
```suggestion
    "name": { "str": "legless heavy ballistic vest with sling", "str_pl": "legless heavy ballistic vests with sling" },
```

File: data/json/items/armor/ballistic_armor.json [hunk 6]
```diff
@@ -439,7 +710,46 @@
...
+    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  This configuration adds shoulder protectors as well as neck protection.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/ballistic_armor.json:730]:
```suggestion
    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  This configuration adds shoulder protectors as well as neck protection.  This one has a single point sling wrapped around the shoulders using a system to keep your weapon extra secure.",
```

File: data/json/items/armor/ballistic_armor.json [hunk 7]
```diff
@@ -485,6 +795,140 @@
       }
     ]
   },
+  {
+    "id": "ballistic_vest_heavy_legs_sling",
+    "type": "ARMOR",
+    "copy-from": "ballistic_vest_heavy_sling",
+    "name": { "str": "sleeveless heavy ballistic vest with sling" },
+    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  This configuration adds groin and neck protection.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/ballistic_armor.json:803]:
```suggestion
    "name": { "str": "sleeveless heavy ballistic vest with sling", "str_pl": "sleeveless heavy ballistic vests with sling" },
    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  This configuration adds groin and neck protection.  This one has a single point sling wrapped around the shoulders using a system to keep your weapon extra secure.",
```

File: data/json/items/armor/ballistic_armor.json [hunk 8]
```diff
@@ -485,6 +795,140 @@
...
+        "specifically_covers": [ "leg_hip_l", "leg_hip_r" ],
+        "volume_encumber_modifier": 0
+      }
+    ]
+  },
+  {
+    "id": "ballistic_vest_heavier_legs",
+    "type": "ARMOR",
+    "copy-from": "ballistic_vest_heavy",
+    "name": { "str": "sleeveless Full Ballistic Protection" },
```
Comment (TheShadowFerret) [inline data/json/items/armor/ballistic_armor.json:838]: >Full
>Sleeveless

🤔 Then it's not full

Also nonsense plural name

File: data/json/items/armor/ballistic_armor.json [hunk 9]
```diff
@@ -485,6 +795,140 @@
...
+        "volume_encumber_modifier": 0
+      }
+    ]
+  },
+  {
+    "id": "ballistic_vest_heavier_legs_sling",
+    "type": "ARMOR",
+    "copy-from": "ballistic_vest_heavy_sling",
+    "name": { "str": "sleeveless Full Ballistic Protection" },
+    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  This configuration adds shoulder protectors, as well as protection for the neck and groin.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/ballistic_armor.json:887]:
```suggestion
    "name": { "str": "sleeveless Full Ballistic Protection" },
    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  This configuration adds shoulder protectors, as well as protection for the neck and groin.  This one has a single point sling wrapped around the shoulders using a system to keep your weapon extra secure.",
```

File: data/json/recipes/armor/torso.json [hunk 10]
```diff
@@ -1196,13 +1210,27 @@
...
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_TORSO",
+    "skill_used": "tailor",
+    "time": "2m 30s",
+    "reversible": true,
+    "decomp_learn": 0,
+    "autolearn": true,
+    "components": [ [ [ "ballistic_vest_heavy", 1 ] ], [ [ "iotv_thigh_plate", 2 ] ] ],
```
Comment (TheShadowFerret) [inline data/json/recipes/armor/torso.json:1231]: This has the same components as the previous added recipe.

**Review by TheShadowFerret — COMMENTED** [review]
I think all the "vests with sling" items ought to be migrated back to the original vest, with a new pocket added to the originals that can only take a single point sling(allowances of the pocket being what the sling's are + the sling's own volume & weight added).

**Comment by Dan1ss1mo** [comment]
Gonna add it as xedra evolved gear after i get a bit more familiar with the mod itself

**Pull request closed without merging**